src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7106,6 +7106,12 @@ def test_aks_nodepool_snapshot(self, resource_group, resource_group_location):
         })
         print("The snapshot resource id %s " % snapshot_resource_id)
 
+        # update tags on nodepool snapshot
+        update_snapshot_cmd = 'aks nodepool snapshot update --resource-group {resource_group} --name {snapshot_name} --tags {tags} -o json'
+        response = self.cmd(update_snapshot_cmd, checks=[
```

Comment:
Where's `tags` defined? And please add some checks on the updated tags

Reply:
Fixed, thanks for catching I'm new to python and this repo 👍